source/qan-intro.rst

```diff
@@ -1,33 +1,233 @@
+
+**Columns**
+
+- The first column is the object's identifier. For *Query*, it is the query's :term:`fingerprint`.
+
+- The second column is the *Main metric*, containing a reduced graphical representation of the metric over time, called a *sparkline*, and a horizontal meter, filled to reflect a percentage of the total value.
+
+- Additional values are revealed as mouse-over tool-tips.
+
+**Tool-tips**
+
+- For the *Query* dimension, hovering over the information icon reveals the query ID and its example.
+
+- Hovering on a column header reveals an informative tool-tip for that column.
+
+- Hovering on the main metric sparkline shows the data value under the cursor.
+
+  .. image:: /_images/PMM_Query_Analytics_Main_Metric_Sparkline.jpg
+
+- Hovering on the main metric meter reveals the percentage of the total, and other details specific to the main metric.
+
+  .. image:: /_images/PMM_Query_Analytics_Main_Metric_Summary.jpg
+
+- Hovering on column values reveals more details on the value. The contents depends on the type of value.
+
+  .. image:: /_images/PMM_Query_Analytics_Metric_Tooltip.jpg
+
+**Adding and removing columns**
+
+- Metrics columns are added with the *Add column* button.
+
+  .. image:: /_images/PMM_Query_Analytics_add-columns.jpg
+
+- When clicked, a text field and list of available metrics are revealed. Select a metric or enter a search string to reduce the list. Selecting a metric adds it to the panel.
+
+- A metric column is removed by clicking on the column heading and selecting *Remove column*.
+
+- The value plotted in the *main metric* column can be changed by clicking a metric column heading and selecting *Swap with main metric*.
+
+**Sorting**
+
+- The entire list is sorted by one of the columns.
+
+- Click either the up or down caret to sort the list by that column's ascending or descending values.
+
+**Pagination**
+
+- Queries are grouped into pages of 10, 50 or 100 items.
+
+- The pagination device lets you move forwards or backwards through pages, jump to a specific page, and choose how many items are listed per page.
+
+  .. image:: /_images/PMM_Query_Analytics_pagination.jpg
+
+.. _pmm-qan-query-selecting:
+.. _query-detail-section:
+
+*************
+Details Panel
+*************
+
+- Selecting an item in the Overview panel opens the *Details panel* with a `Details Tab`_.
+
+- If the dimension is *Query*, the panel also contains the `Examples Tab`_, `Explain Tab`_, and `Tables Tab`_.
+
+===========
+Details Tab
+===========
+
+The *Details* tab contains a *Query time distribution* bar (only for MySQL databases)
+and a set of *Metrics* in collapsable subpanels.
+
+.. image:: /_images/PMM_Query_Analytics_details.jpg
+
+- The *Query time distribution* bar shows a query's total time made up of colored segments, each segment representing the proportion of time spent on one of the follow named activities:
+
+  - ``query_time``: Elapsed time between receiving a request and the returned result.
+  - ``lock_time``
```

Comment:
@lunaticusgreen Where can I find descriptions for these?

Reply:
@PaulJacobs-percona https://github.com/percona/grafana-dashboards/blob/e348eefe9cd03cae8a9e5c10e18dc5760cee276a/pmm-app/src/pmm-qan/panel/QueryAnalytics.constants.ts#L41 PTAL here, all descriptions that we have on a frontend